.gitignore

```diff
@@ -4,3 +4,4 @@
 /backdrop/
 /multisite/
 /backdrop.zip
+.idea/
```

Comment:
Remove this please - I think this must be from your local

Reply:
I am using PHPStorm, so Any commit wants the .idea folder, how do I get round that?